tests/nosetests/pyanaconda_tests/localization_test.py

```diff
@@ -17,9 +17,191 @@
 #
 
 from pyanaconda import localization
+from pyanaconda.core.constants import DEFAULT_LANG
 from pyanaconda.core.util import execWithCaptureBinary
 import locale as locale_mod
 import unittest
+from unittest.mock import call, patch, MagicMock
+from io import StringIO
+
+
+class LangcodeLocaleParsingTests(unittest.TestCase):
+    def is_valid_test(self):
```

Comment:
You should also have an invalid test case.

Reply:
Will do. The name is a bit misleading because the names are taken from the functions tested, so there won't be a new is_invalid_test ...

Reply:
Added.